contrib/database/sql/sql_test.go

```diff
@@ -295,11 +348,14 @@ func TestRegister(_ *testing.T) {
 }
 
 func TestNamingSchema(t *testing.T) {
-	newGenSpansFunc := func(t *testing.T, driverName string) namingschematest.GenSpansFn {
+	newGenSpansFunc := func(t *testing.T, driverName string, registerOverride string) namingschematest.GenSpansFn {
```

Comment:
Can `registerOverride` be a bool ? It seems like it's either `register-override` or empty string `""`

Reply:
I think it's a good suggestion, however I've set it as a `string` to keep it consistent with the `serviceOverride string` option I've been adding to all these tests. The reason I used a string originally is because later this value is used for the test assertions, so I thought it would preferable to give control to the caller so they know what value to assert for.

I don't have a strong preference for one or the other, but if possible I would like to keep it consistent with the rest of the `TestNamingSchema` tests from other packages.